make better use of SFX on SelectMusicCasual

quietly-turning opened the pull request:
# About

ScreenSelectMusicCasual wasn't using very many sound effects previously.  

For example, no sound effects would be played when interacting with the custom PlayerOptions menu (arrow spacing, difficulty, and confirming song choice).

Since Casual Mode is intended for genuine novices—casual players at a public arcade or anime convention who don't know anything about StepMania or DDR and just want to play a game with a friend with as little UI friction as possible—I think it should focus on streamlining players into gameplay.   

Part of this means cueing novices that *something has happened* when they press cabinet buttons, and one established convention for conveying that is using sound effects. 🙂 
 
 ## Changes
 
This commit makes use of three existing sound effects files as ActorSounds:

 * **ScreenSelectMaster change.ogg** - for scrolling between groups
 * **Common start.ogg** - for confirming song choice
 * **MusicWheel change.ogg** - for scrolling between songs and changing modifiers

## Technical Notes

The ActorSounds each have `IsAction` set to true, allowing machine operators to mute them at their discretion using the `MuteActions` StepMania preference (can be toggled via <kbd>F3</kbd> + <kbd>A</kbd>).

natano commented:
Makes sense to me! One nitpick: I think it would make sense to also add a sound for closing a folder, maybe the same sound you play when selecting a folder, or the one played when switching between folders.

quietly-turning committed:
make better use of SFX on SelectMusicCasual

ScreenSelectMusic wasn't using very many sound effects previously.
For example, no sound effects would be played when interacting with the
custom PlayerOptions menu (arrow spacing, difficulty, and confirming
song choice).

This commit uses three existing sound effects files as ActorSounds:

 • ScreenSelectMaster change.ogg - for scrolling between groups and
                           closing a currently-open group folder
 • Common start.ogg - for confirming song choice
 • MusicWheel change.ogg - for scrolling between songs and changing
                           modifiers

The ActorSounds each have "IsAction" set to true, allowing machine
operators to mute them at their discretion using the MuteActions
StepMania preference (can be toggled via F3 + A).

quietly-turning commented:
> I think it would make sense to also add a sound for closing a folder

I agree!  Thanks for catching this and pointing it out.

I've updated the commit to use the existing **ScreenSelectMaster change.ogg** for closing a folder.  It has a "whoosh" sound, like something moving through air, and fits the current *Close Folder* animation for SelectMusicCasual well enough.
